index.html

```diff
@@ -0,0 +1,268 @@
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Three -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button class="green-button">See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Four -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card five -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+          <!-- Project Card Six -->
+          <div class="projects">
+            <div class="project-image"></div>
+            <div class="project-description">
+              <h4>
+                Multi-Post Stories <br />
+                Gain+Glory
+              </h4>
+              <ul class="languages">
+                <li>Ruby on rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+              </ul>
+              <button>See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- About Section -->
+      <section id="about-section">
+        <h4 id="about-intro">About me</h4>
+        <p id="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <div><button>Get My Resume</button></div>
```

Comment:
## Optional but recommended
The button should always have a type, to describe what the button is doing or is supposed to do. Why, read [here](w3schools.com/tags/att_button_type.asp)